tests/Tests/SkiaSharp/SKDrawableTest.cs

```diff
@@ -6,6 +6,41 @@ public class SKDrawableTest : SKTest
 	{
 		[SkippableFact]
 		public void CanInstantiateDrawable()
+		{
+			using (var drawable = new TestDrawable())
+			{
+			}
+		}
+
+		[SkippableFact]
+		public void CanAccessBounds()
+		{
+			using (var drawable = new TestDrawable())
+			{
+				Assert.Equal(SKRect.Create(100, 100), drawable.Bounds);
+				Assert.Equal(1, drawable.BoundsFireCount);
+			}
+		}
+
+		[Trait(Traits.FailingOn.Key, Traits.FailingOn.Values.macOS)] // Something with sk_sp<SkPicture> SkDrawable::onMakePictureSnapshot() is causing issues
+		[Trait(Traits.FailingOn.Key, Traits.FailingOn.Values.iOS)] // Something with sk_sp<SkPicture> SkDrawable::onMakePictureSnapshot() is causing issues
+		[Trait(Traits.FailingOn.Key, Traits.FailingOn.Values.MacCatalyst)] // Something with sk_sp<SkPicture> SkDrawable::onMakePictureSnapshot() is causing issues
```

Comment:
I notice that the fact I override the 

```cpp
sk_sp<SkPicture> SkDrawable::onMakePictureSnapshot()
```

function in C++ for the managed drawable is causing a crash. If i change it to

```cpp
SkPicture* SkDrawable::onMakePictureSnapshot()
```

stops the crash. I noticed this locally on macOS. Not sure why it passes on CI.